### PR TITLE
Material UI - Login

### PR DESCRIPTION
**Pull request by cabarnes**
Converts the login screen to Material UI

### CODE REVIEW

**Comment by aninternetof** [comment]
Is there a way to center the social auth buttons within their allotted column space?

image

File: src/containers/Accounts/Login.js [hunk 1]
```diff
@@ -108,34 +116,54 @@ class Login extends Component {
       defaultMessage: 'Password',
     });
 
+    const GoogleButton = withStyles((theme) => ({
```
Comment (aninternetof) [inline src/containers/Accounts/Login.js:119]: I wonder how these hooks worked for you but threw an error for me. I thought they weren't supposed to work in class-type components.

Comment (aninternetof) [reply src/containers/Accounts/Login.js:119]: Whatever -- now I can just copy this!

**Comment by cabarnes** [comment]
@aninternetof I have tried seemingly every combination of `justify` and `alignItems` for that grid container and the buttons never change location. Maybe because they are `withStyles`? If you have any other ideas of what to try or want to try yourself, let me know. Otherwise, maybe doing more screens will help to learn better how to lay this out.

**Comment by cabarnes** [comment]
@aninternetof I believe it's working correctly now. I was getting some weird caching issues but this seems to have centered the buttons.

**Pull request merged**